scripts/fetch_and_build_rpm

```diff
@@ -173,38 +173,42 @@ sed -i -E -e "\\|^Source0:|s|https:.*|${tarballurl}|" \
     "${builddir}/${pkgname}.spec"
 
 osname=$(awk '{print $1}' /etc/system-release)
+os_release=$(awk '{print $4}' /etc/system-release)
```

Comment:
Nit, can we rename this to `osrelease` so we are consistent with the naming of `osname` ?

Reply:
Changed osname into os_name